puppet-client.html

```diff
@@ -239,7 +232,8 @@
             }
         },
         onLocalChange: function(patches) {
-            this.propagateDomBind(patches);
+            // todo #26
+            this.propagateDomBind({patches: patches, results: patches.map(function() { return {};})});
```

Comment:
Are you sure that none of those patches removed a thing? Isn't it an overhead to produce an array of empty objects?